### ISSUE TITLE
Different format for icon build config names

### ISSUE THREAD

**Issue by bgentry** [FortAwesome/ember-fontawesome#29]
When using an icon, you do this:

```hbs
{{fa-icon "ambulance"}}
```

However, when specifying an icon to include in `ember-cli-build.js`, you do this:

```js
[ "faAmbulance" ]
```

Do these need to be different? Can the build file just use `ambulance` and do the casing conversion on its own?

**Comment by mlwilkerson** [comment]
I think that's reasonable, yes. I think we've gotten used to switching back and forth between the `iconName` (just "ambulance") and the JS name by which these normally appear in our code ( "faAmbulance"), so I hardly notice any more. But I agree that this is something that is confusing, especially for newcomers—I remember that.

However, given that this is an issue that runs deeper, while avoiding it in `ember-cli-build.js`, one might still bump into elsewhere, such as if doing named imports of icons. This one, for example, would be exported as `faAmbulance`. So you'd have to import it as such, like 

See for example [here in the template](https://github.com/FortAwesome/ember-fontawesome/blob/942488d780e927a901f5aab613ce47d0d1756166/tests/dummy/app/templates/application.hbs#L3) of the dummy app. And [here where that icon is imported](https://github.com/FortAwesome/ember-fontawesome/blob/942488d780e927a901f5aab613ce47d0d1756166/tests/dummy/app/controllers/application.js#L2).

We have a similar challenge with CSS class naming, in fact. If you weren't using this Ember addon and just specifying the icon via class names on an `<i>` tag, for example, you'd have to reference the icon like this: `<i class="fas fa-ambulance"></i>`. So which is it? `ambulance`, `faAmbulance`, or `fa-ambulance`? At the moment, it depends upon context. So changing this API in the `ember-cli-build.js` isn't going to get rid of that problem entirely (the possible confusion that the way you reference an icon may differ by context), but it might still be a preferable way to do it in that case.

So, having pointed that out, does it still seem preferable to use the `iconName` instead of the JS name in `ember-cli-build.js`?

**Comment by bgentry** [comment]
I should have picked a hyphenated example instead of a one-word :)

For an icon like `conveyer-belt-alt`, I wouldn't see much as much benefit in having to specify `conveyerBeltAlt` instead of `faConveyerBeltAlt`. To me the real user benefit would come from using the exact same thing in the helper and the build config.

As for JS imports, it's customary is to import camel-cased names. I think this is a *much* rarer use case than customizing the build config, so I don't see much of an issue with:

```js
import { faConveyerBeltAlt } from '@fortawesome/fontawesome-pro-light'
```

Basically the `fa` prefix in pure JS doesn't bother me much. I realize that the build step is basically just importing these directly, so perhaps it's because thus far I haven't used the JS imports on their own. Maybe it's a question of how common the build config will be vs the pure JS imports?

**Comment by jrjohnson** [comment]
The build config will be used by almost everyone. Even when #21 is done and no config is needed for static icons it will still be necessary to use it for dynamic ones.  Importing the icon in JS and using it in the template is going to be very rare.

It makes sense to me that the non `fa` prefixed version would be used everywhere and that the JS import is unique and that's OK.
